### ISSUE TITLE
Exclude elements

### ISSUE THREAD

**Issue by LostCrew** [blasten/turn.js#434]
hello @blasten,

wouldn't it be nice to have a `exclude` option to let the user specify elements inside the turn container that have not to be transformed and treated as page?
alternatively, if you prefer, it could also be `elements` or `items` and behaves in the opposite way.

this parameter could accept a DOM element, a well-formed jQuery selector, a jQuery object or a function that returns any of the latter value.

a concrete case: i need to have some extra elements with `position: absolute` inside the book and rely on its `position: relative`, page placement and size to accurately position them. my current workaround is: write them outside the book and, only after turnjs has been initalized, execute a relocation inside the book with `.append()`.

what do you think?

P.S: besides, it would also be delicious to leverage on a new event fired when the initialization of the book is completed, something like `initialized` or `init`.
